### ISSUE TITLE
Onboarding - campus access instructions unclear

### ISSUE THREAD

**Issue by hermancollin** [neuropoly/intranet.neuro.polymtl.ca#71]
In the deprecated onboarding google doc, the *campus access* section said this:
> To get access to campus, you will need to have an appointment booked for you with https://www.polymtl.ca/surete/horaire-et-acces to have a photo taken and a card printed.
> **Special COVID rules**: see #covid-19 on Slack. Follow the training there and then get yourself inscribed with Sûreté by Julien Cohen-Adad.

This part is missing from the lab manual. [Here](https://intranet.neuro.polymtl.ca/onboarding/campus-access.html#requirements-for-room-access), it says this:
> To be able to access the lab rooms, you will need a Polytechnique card and to activate it for the specific rooms you wish to access. To activate it, please follow the onboarding procedure.

But the room access is never mentioned again in the onboarding section.
I already had a card and I went to the sureté desk in the main building. They told me to ask my supervisor for room access and they gave me a card that said this:
> Demande d'accès
> Seul (sic) la personne attitrée à votre département peut demander un accès. Un courriel doit être envoyé à: sur-access@polymtl.ca

Maybe this should be mentioned in the lab manual more explicitly.

**Comment by kousu** [comment]
I agree. I dislike the entire /onboarding section in its current state.

What happened is that last summer we started trying to improve it, but then realized we actually didn't have a process at all. The onboarding process [ended up getting](https://github.com/neuropoly/onboarding/issues/30) written up [here](https://github.com/neuropoly/onboarding/blob/cfe236dcff7e0c54aba7529e1fb3a1c29e82f99f/.github/ISSUE_TEMPLATE/membership-card.md).

I would like to sync it back. I think it lives more naturally in the manual, and is more helpful there since prospective students can see the process transparently there too. It's just an annoying bit of admin work that no one has taken on yet. Unfortunately onboarding is about more than just getting accounts, too, which is what that issue template focuses on, it's about getting oriented to the city, the tax laws, landlords, etc. Moreover some of this is currently [over here in a different subfolder](https://github.com/neuropoly/intranet.neuro.polymtl.ca/blob/3ac5ab2da9b967a6ecb67e6ef2a8b5b1303e3405/practical-information/moving-to-montreal.md) and it's not clear which folder should win out.
